db/schema.rb

```diff
@@ -26,15 +26,15 @@
   end
 
   create_table "delayed_jobs", force: :cascade do |t|
-    t.integer  "priority",               default: 0, null: false
-    t.integer  "attempts",               default: 0, null: false
-    t.text     "handler",                            null: false
+    t.integer  "priority",   default: 0, null: false
```

Comment:
What's up with all the changes in this file? It looks like all they do is remove field size limits, which doesn't seem necessary given the task at hand...